Source/CTPanoramaView.swift

```diff
@@ -230,8 +232,9 @@ import ImageIO
     }
     
     private func resetCameraAngles() {
-        cameraNode.eulerAngles = SCNVector3Make(0, 0, 0)
-        self.reportMovement(0, xFov.toRadians(), callHandler: false)
+        let y = (self.startAngleDegrees * .pi) / 180
```

Comment:
The usage of `self` is unnecessary. Also there's a method named `toRadians()`  in this file so there's no need to manually convert to radians like you did here.